docs/source/ns_blog.md

```diff
@@ -0,0 +1,139 @@
+
+
+
+## What is Neural Solution?
+
+Neural Solution provide task-level and tuning-level parallelism, coordinating the optimization task queue and leveraging distributed tuning to speedup optimization process. It also offers a seamless integration interface, eliminating the need for repetitive environment setups and code adaptation, simplifying the optimization process for users.
+
+Neural Solution efficiently schedules the optimization task queue by coordinating available resources and tracking the execution status of each task. This concurrent scheduling ensures optimal resource utilization and allows for efficient execution of multiple optimization tasks simultaneously.
+
+One major challenge in model quantization is identifying the optimal accuracy-relative configuration which is time-consuming. To mitigate this pain point, Neural Solution allows users to parallelize the tuning process across multiple nodes by simply specifying the number of workers in the task request.
+
+In addition, Neural solution also offers a convenient interface for seamless integration into different applications or platforms. It exposes both RESTful and gRPC APIs, empowering users to submit quantization tasks, query the optimization process, and obtain tuning results with ease.
+
+Moreover, for the Hugging Face models, Neural Solution eliminates the need for any code modifications during the optimization process by seamlessly integrating the functionality of the Neural Coder. This approach significantly lowers the barrier to entry for users who may not possess extensive coding expertise.
+
+
+![NS-OaaS-Intro (1)](../../neural_solution/docs/source/imgs/NS-OaaS-Intro.png)
+
+Fig 1. how does the neural solution work
+
+
+
+## Get started with neural solution
+
+Let's get start Neural Solution with an end-to-end example that quantizes a [Text classification model](https://github.com/huggingface/transformers/tree/v4.21-release/examples/pytorch/text-classification) from Hugging Face.
+
+### Install neural solution
+
+```shell
+# get source code
+git clone https://github.com/intel/neural-compressor
+cd neural-compressor
+
+# install neural solution
+pip install -r neural_solution/requirements.txt
+python setup.py neural_solution install
+```
+
+> More installation options and details can be found [here](https://github.com/intel/neural-compressor/tree/master/neural_solution#installation).
+
+### Start the Neural Solution Service
+
+```shell
+# Start neural solution service with default configuration, log will be saved in the "serve_log" folder.
+neural_solution start
+
+# Start neural solution service with custom configuration
+neural_solution start --task_monitor_port=22222 --result_monitor_port=33333 --restful_api_port=8001
```

Comment:
How about adding successful startup output information?
Let users know the submission entry point.